### PR TITLE
feat(goctl):Add ignore-columns flag

### PR DESCRIPTION
**Pull request by anqiansong**
Added `--ignore-columns, -i` flag to specify the columns that need to be ignored when inserting or updating rows, for example: you need to ignore the `create_time` and `update_time` columns, you can use `--ignore-columns, -i` flag for model generation command reference is as follows

```
# generated from sql file
$ goctl model mysql ddl --src $sql_file --dir $output -i 'create_time' -i 'update_time'
# or use comma splitter
# $ goctl model mysql ddl --src $sql_file --dir $output -i 'create_time,update_time'

# generated from dsn
$ goctl model mysql datasource --url $dsn --table $patterns --dir $output -i 'create_time' -i 'update_time'
# or use comma splitter
# $ goctl model mysql datasource --url $dsn --table $patterns --dir $output -i 'create_time,update_time'
```

### CODE REVIEW

File: tools/goctl/model/sql/gen/vars.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package gen
 
 import (
+	"fmt"
+	"github.com/zeromicro/go-zero/core/collection"
```
Comment (kevwan) [inline tools/goctl/model/sql/gen/vars.go:5]: group imports.

**Comment by kevwan** [comment]
Is it better to support `-i 'create_time,update_time'`?

**Comment by anqiansong** [comment]
> Is it better to support `-i 'create_time,update_time'`?

Done

**Comment by yogibaba** [comment]
Execution on PostgreSQL, gives `Error: unknown flag: --ignore-columns`

**Pull request merged**